Makefile

```diff
@@ -3,13 +3,13 @@ SHELL = /bin/sh
 # --args:
 #   diAccessLevel - access level of the dependency injection graph. If module is used by another module (like Cio) then you want `public`. Else, `internal`. 
 #   moduleName - the name of the module that you are generating code for. 
-#   imports - Import statements to be at the top of the generated files in case the file needs classes from other modules. Split by `:` (example: `imports=Cio-Foo-Bar`) 
+#   imports - Import statements to be at the top of the generated files in case the file needs classes from other modules. Split by `-` (example: `imports=Cio-Foo-Bar`) 
 generate:
-	sourcery --sources Sources/Common --templates Sources/Templates --output Sources/Common/autogenerated --args diAccessLevel=public,moduleName=Common
-	sourcery --sources Sources/Tracking --templates Sources/Templates --output Sources/Tracking/autogenerated --args diAccessLevel=internal,moduleName=Tracking,imports=Common
-	sourcery --sources Sources/MessagingPush --templates Sources/Templates --output Sources/MessagingPush/autogenerated --args diAccessLevel=public,moduleName=MessagingPush,imports=CioTracking:Common
-	sourcery --sources Sources/MessagingPushAPN --templates Sources/Templates --output Sources/MessagingPushAPN/autogenerated --args diAccessLevel=internal,moduleName=MessagingPushAPN,imports=CioMessagingPush:CioTracking:Common
-	sourcery --sources Sources/MessagingPushFCM --templates Sources/Templates --output Sources/MessagingPushFCM/autogenerated --args diAccessLevel=internal,moduleName=MessagingPushFCM,imports=CioMessagingPush:CioTracking:Common
+	sourcery --sources Sources/Common --templates Sources/Templates --output Sources/Common/autogenerated 
+	sourcery --sources Sources/Tracking --templates Sources/Templates --output Sources/Tracking/autogenerated --args imports=Common
+	sourcery --sources Sources/MessagingPush --templates Sources/Templates --output Sources/MessagingPush/autogenerated --args imports=CioTracking-Common
+	sourcery --sources Sources/MessagingPushAPN --templates Sources/Templates --output Sources/MessagingPushAPN/autogenerated --args imports=CioMessagingPush-CioTracking-Common
+	sourcery --sources Sources/MessagingPushFCM --templates Sources/Templates --output Sources/MessagingPushFCM/autogenerated --args imports=CioMessagingPush-CioTracking-Common
```

Comment:
Bonus: We got to greatly simplify the source code generation code for the DI graph 🎊